### DISCUSSION TITLE
How to use github

### DISCUSSION THREAD

**Discussion by arafat425** [community/community#28369] [New to GitHub]
how to use?

**Comment by jfullstackdev** [comment]
GitHub is a centralized space for code, you can think of it as a storage but with all the functionalities for collaboration, that was based on Git.

But as more and more features are being released, it's being used innovatively for other purposes. For example, GitHub Pages was intended for Project's Documentation, yet you can host your simple site using it, this GitHub Discussions, you can use as a forum even for non-tech topics as long as you don't violate any rules. The new Projects is a great way to organize your tasks.

for more info on how to get started on GitHub, check this out

https://docs.github.com/en/get-started
